pkg/machinery/config/types/block/user_volume_config.go

```diff
@@ -68,6 +73,14 @@ type UserVolumeConfigV1Alpha1 struct {
 	//     lowercase and uppercase ASCII letters, digits, and hyphens.
 	MetaName string `yaml:"name"`
 	//   description: |
+	//     Volume type.
+	//   values:
+	//     - partition
+	//     - bind
```

Comment:
that would be less confusing for users:

```suggestion
	//     - directory
```